tensorly/backend/tensorflow_backend.py

```diff
@@ -198,8 +198,12 @@ def log2(self,x):
     (tf.einsum, 'einsum'),
     (tf.tensordot, 'tensordot'),
     (tfm.sin, 'sin'),
+<<<<<<< HEAD
+    (tfm.cos, 'cos')
+=======
```

Comment:
Looks like there was a merge conflict that wasn't resolved - here the lines below should be kept, adding cos and anu.